### PR TITLE
converting global config extension to dynamic-plugin-sdk

### PR DESCRIPTION
**Pull request by glekner**
depends on #7919

### CODE REVIEW

File: frontend/packages/console-dynamic-plugin-sdk/src/extensions/cluster-settings.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { Extension } from '@console/plugin-sdk/src/typings/base';
+
+export namespace ExtensionProperties {
+  export type ClusterGlobalConfig = {
+    /** Unique identifier for the cluster config resource instance. */
+    uid: string;
```
Comment (christianvogt) [inline frontend/packages/console-dynamic-plugin-sdk/src/extensions/cluster-settings.ts:6]: While we currently use `uid` in the static extension, we should update to `id` to be consistent with other extensions where `id` denotes a unique identifier.

**Comment by glekner** [comment]
please review @christianvogt

**Comment by rawagner** [comment]
/hold
compilation error

```
ERROR in /go/src/github.com/openshift/console/frontend/public/components/cluster-settings/global-config.tsx
ERROR in /go/src/github.com/openshift/console/frontend/public/components/cluster-settings/global-config.tsx(105,22):
TS2339: Property 'id' does not exist on type 'ClusterGlobalConfig | { model: any; uid: any; name: any; namespace: any; kind: any; }'.
  Property 'id' does not exist on type '{ model: any; uid: any; name: any; namespace: any; kind: any; }'.
```

**Comment by christianvogt** [comment]
@glekner ^^

**Comment by glekner** [comment]
fixed @rawagner @christianvogt

**Pull request merged**